src/components/Circular Support Indicator/variables.scss

```diff
@@ -0,0 +1,39 @@
+/* ----------------------------------------------
+* abstracts/_variables.scss
```

Comment:
Would be good idea to use these variables through all storybook, not just in this component. What do you think @tobiastykvart? And yes, we need to agree how to work with styling and then think how to use variables.

Reply:
I started to look into this before going on holidays, I'll continue to set that up. I think most variables will be in the MUI theme and also accessible, not just for MUI components.